src/Driver/DriverFactory.php

```diff
@@ -87,18 +95,29 @@ protected static function getPeclConnection(array $connection)
     /**
      * @param string[] $connection
      *
-     * @return AMQPLazyConnection
+     * @return AMQPStreamConnection
      *
      * @codeCoverageIgnore
      */
     protected static function getPhpAmqpLibConnection(array $connection)
     {
-        return new AMQPLazyConnection(
-            $connection['host'],
-            $connection['port'],
-            $connection['user'],
-            $connection['pwd']
-        );
+        if (isset($connection['ssl']) && $connection['ssl']) {
```

Comment:
What is the reason of checking `$connection['ssl']` ?

Reply:
I wanted to keep the previous behaviour : no `ssl` config means no `ssl`
I also wanted to provide a way to config writers to deactivate `ssl` by setting it to false.
You see another way?

Reply:
>I also wanted to provide a way to config writers to deactivate ssl by setting it to false.

Oh I see. 👍